.prettierrc

```diff
@@ -3,5 +3,6 @@
 	"singleQuote": true,
 	"tabWidth": 4,
 	"trailingComma": "none",
-	"useTabs": true
+	"useTabs": true,
+	"printWidth": 90
```

Comment:
`"printWidth": 90` should be listed at the top of the file, to keep it alphabetised.

Reply:
yeah... good point @Alhadis, BTW will push the formatted later. if merged